core/src/main/scala/cats/Traverse.scala

```diff
@@ -37,6 +37,26 @@ import scala.annotation.implicitNotFound
    */
   def traverse[G[_]: Applicative, A, B](fa: F[A])(f: A => G[B]): G[F[B]]
 
+  /**
+   * Given a function which returns a G effect, thread this effect
+   * through the running of this function on all the values in F,
+   * returning an F[A] in a G context, ignoring the values
+   * returned by provided function.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
```

Comment:
Double import (counts as unused on 2.13, which is why I think the build failed on that version)

Reply:
fixed now